src/hooks/persisted/useNovel/store-helper/bootstrapService.ts

```
@@ -0,0 +1,334 @@
+    pageIndex,
+    settingsSort,
+    settingsFilter,
+  }: {
+    novel: NovelInfo | undefined;
+    novelPath: string;
+    pluginId: string;
+    pageIndex: number;
+    settingsSort: ChapterOrderKey;
+    settingsFilter: ChapterFilterKey[];
+  }): Promise<BootstrapResult> => {
+    console.time(`bootstrap_${pluginId}_${novelPath}`);
```

ESLint

[error] Unexpected console statement.

(no-console)